README.md

```diff
@@ -355,6 +355,7 @@ nginx_main_template:
     cache: false
     rate_limit: false
     keyval: false
+    #server_tokens: "off"
```

Comment:
You don't need to enclose `off` in double quotes
```suggestion
    #server_tokens: off
```

Reply:
No, I'm needed it:

```
server_tokens: off # will render as => server_tokens: False
```

Reply:
Oh interesting. I didn't know about that behaviour.